scripts/jest/setupHostConfigs.js

```diff
@@ -14,11 +14,6 @@ function resolveEntryFork(resolvedEntry, isFBBundle) {
   // .js
 
   if (isFBBundle) {
-    if (__EXPERIMENTAL__) {
-      // We can't currently use the true modern entry point because too many tests fail.
-      // TODO: Fix tests to not use ReactDOM.render or gate them. Then we can remove this.
-      return resolvedEntry;
-    }
```

Comment:
This is key in getting the modern www build to use it's own entrypoint which is needed to make tests faithfully use the APIs included in this entrypoint rather than what is available on the unsuffixed file.